.gitattributes

```diff
@@ -0,0 +1,3 @@
+Gemfile.lock linguist-generated=true
+yarn.lock linguist-generated=true
+.rubocop_todo.yml linguist-generated=true
```

Comment:
https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

We don't need to be reviewing generated files.